examples/quickstart.py

```diff
@@ -0,0 +1,54 @@
+import pathlib
```

Comment:
I like this script! It really emphasizes how similar the interfaces of the different algorithms are (for simple use cases anyway) which is great.

Maybe add a one-liner module docstring in case someone just stumbles across this script without seeing the README?